server/container/guice/cassandra-guice/src/test/java/org/apache/james/CassandraTaskManagerTest.java

```diff
@@ -47,10 +47,10 @@
 import org.apache.james.task.eventsourcing.EventSourcingTaskManager;
 import org.apache.james.task.eventsourcing.TaskExecutionDetailsProjection;
 import org.apache.james.task.eventsourcing.WorkerStatusListener;
-import org.apache.james.task.eventsourcing.cassandra.CassandraTaskExecutionDetailsProjection;
-import org.apache.james.task.eventsourcing.cassandra.CassandraTaskExecutionDetailsProjectionDAO;
-import org.apache.james.task.eventsourcing.cassandra.CassandraTaskExecutionDetailsProjectionModule;
```

Comment:
I don't agree with that: putting Cassandra classes into cassandra package still makes sense.